CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+*   Release Cloud Firestore emulator v1.14.0
+    *   Chore: prepare for aggregate support.
```

Comment:
Is this a user-facing change? If not, please remove.

Reply:
Done.